dmutils/apiclient/search.py

```diff
@@ -5,13 +5,29 @@
 
 
 class SearchAPIClient(BaseAPIClient):
+    def __init__(self, *args, **kwargs):
+        self.index_name = kwargs.pop('index_name', 'g-cloud')
```

Comment:
What do you mean? Is there an issue caused by adding a keyword argument to the constructor or is this about how we  don't want to create a separate client for each index?

I think we should definitely add index as a method argument where it will make sense, I'm trying to avoid this for now since I don't really need multi-index clients for scripts and constructor argument seems like it's less likely to break compatibility.